### PR TITLE
#151991576 Update README with status badges

### PR DESCRIPTION
**Pull request by arnawldo**
#### What does this PR do?
Add status badges from Travis CI, Coveralls.io, and Codacy to project
#### Description of Task to be completed?
Setup automatic builds by Travis CI, coverage reporting from coveralls.io, and code quality inspection from Codacy
#### How should this be manually tested?
Click each badge on the README file of the repo and confirm that the links lead to the respective third party sites with correct information on the project
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#151991576 . 
#151991421
#### Questions:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+[![Build Status](https://travis-ci.org/andela/healthchecks-apollo.svg?branch=master)](https://travis-ci.org/andela/healthchecks-apollo)
+[![Coverage Status](https://coveralls.io/repos/github/andela/healthchecks-apollo/badge.svg?branch=master)](https://coveralls.io/github/andela/healthchecks-apollo?branch=master)
```
Comment (ivantoz) [inline README.md]: Same here

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+[![Build Status](https://travis-ci.org/andela/healthchecks-apollo.svg?branch=master)](https://travis-ci.org/andela/healthchecks-apollo)
```
Comment (ivantoz) [inline README.md]: Have the badge point to develop branch

**Review by ivantoz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (arnawldo) [reply README.md]: On it now

File: README.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+[![Build Status](https://travis-ci.org/andela/healthchecks-apollo.svg?branch=develop)](https://travis-ci.org/andela/healthchecks-apollo)
+[![Coverage Status](https://coveralls.io/repos/github/andela/healthchecks-apollo/badge.svg?branch=master)](https://coveralls.io/github/andela/healthchecks-apollo?branch=develop)
```
Comment (ivantoz) [inline README.md]: Can you change this to point to the correct branch

**Review by ivantoz — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Arthur236) [reply README.md]: Getting right on it

**Review by ivantoz — APPROVED** [review]

**Pull request merged**